### PR TITLE
Updated application gradient utility classes and added storybook documentation

### LINKED ISSUE #492: Update utility classes for background gradients in Applications
@praveen-murali-ind _a

Screenshot 2021-10-04 at 5 46 52 PM

### PR DESCRIPTION
**Pull request by praveen-murali-ind**
Fixes #492 

Added documentation for Application Icons and Gradient utility classes.

Screenshot 2021-10-05 at 11 00 41 AM

@karthiknmenon _a Please review.

//cc @edwinbbu

### CODE REVIEW

File: stories/Iconography.stories.jsx [hunk 1]
```diff
@@ -38,13 +55,14 @@ export const Iconography = () => {
           <span>Source</span>
         </a>
       </p>
-      <div className="grid grid-cols-4 lg:grid-cols-8 gap-3">
+      <div className="grid grid-cols-4 gap-3 lg:grid-cols-8">
+        {console.log(iconset.iconList)}
```
Comment (karthiknmenon) [inline stories/Iconography.stories.jsx:59]: Please remove this console statement here.

**Review by karthiknmenon — COMMENTED** [review]
@praveen-murali-ind _a Minor suggestion. The rest looks good.

File: stories/Iconography.stories.jsx (hunk 1 above)
Comment (praveen-murali-ind) [reply stories/Iconography.stories.jsx]: > Please remove this console statement here.

Done.

**Review by karthiknmenon — APPROVED** [review]
LGTM.

**Pull request merged**